src/App.jsx

```diff
@@ -14,70 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
```

Comment:
For the default value, it is better to refer directly to the array of `goods` to understand that we will always have relevant information.
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods[8]);
```